docs/modules/nifi/pages/troubleshooting/index.adoc

```diff
@@ -21,6 +21,29 @@ spec:
 
 == `HTTP ERROR 400 Invalid SNI`
 
+=== Local PORT-FORWARD
```

Comment:
Can we put this below the `=== NGINX ingress-controller` one? I think this is less important.

Reply:
removed